lib/page/home/home_page.dart

```diff
@@ -49,10 +57,8 @@ class _HomePageState extends ConsumerState<HomePage> {
           cacheLoadingSuccess: () => _buildHomePage(surveys),
           apiLoadingSuccess: () =>
               _buildHomePage(surveys, shouldEnablePagination: true),
-          error: (exception) {
-            WidgetsBinding.instance.addPostFrameCallback((_) {
-              _showError(AppLocalizations.of(context)!.homeLoadSurveysError);
-            });
+          error: () {
+            _showError(AppLocalizations.of(context)!.homeLoadSurveysError);
```

Comment:
That being said, we should try to match the exception to the string and show it before showing the generic one. What do you think?

```suggestion
            _showError(exception.getErrorMessage() ?? AppLocalizations.of(context)!.homeLoadSurveysError);
```